ui/src/components/form/form.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+import { Form } from './form';
+
+export default {
+  title: 'Components/Form',
+  component: Form,
+};
+
+export const Fields = () => {
+  const [file, setFile] = useState<File | null>(null);
+  return (
+    <>
+      <Form.Field>
+        <Form.Label>Label</Form.Label>
+        <Form.Input placeholder="Input" />
+        <Form.Error>Input error</Form.Error>
+      </Form.Field>
+      <Form.Field>
+        <Form.Label>Label</Form.Label>
+        <Form.Textarea placeholder="Textarea" />
+        <Form.Error>Textarea error</Form.Error>
+      </Form.Field>
+      <Form.Field css={{ width: '$24' }}>
+        <Form.Label>Label</Form.Label>
+        <Form.File value={file} onChange={(file) => setFile(file)} />

```

Comment:
This is broken in storybook:
<img width="240" alt="image" src="https://user-images.githubusercontent.com/30053103/215077072-d28ac47d-f2dd-441a-9f75-28f0bd5ca176.png">